pom.xml

```diff
@@ -699,15 +699,10 @@
               <artifactId>checkstyle</artifactId>
               <version>${puppycrawl.checkstyle.version}</version>
             </dependency>
-            <dependency>
-              <groupId>org.apache.bookkeeper</groupId>
-              <artifactId>buildtools</artifactId>
-              <version>${project.version}</version>
-            </dependency>
           </dependencies>
           <configuration>
-            <configLocation>bookkeeper/checkstyle.xml</configLocation>
-            <suppressionsLocation>bookkeeper/suppressions.xml</suppressionsLocation>
+            <configLocation>buildtools/src/main/resources/bookkeeper/checkstyle.xml</configLocation>
```

Comment:
can you explain why this change is needed? if it is just a version problem, why does this work in 4.8.0, not working in 4.9.0?

Reply:
The checkstyle plugin has a dependency on org.apache.bookeeper:buildtools:${version}. If you bump the version and run checkstyle without running mvn install first, like we do in CI, CI will fail because it can't find the dependency.